code/modules/mob/living/carbon/human/species_types/moth.dm

```diff
@@ -0,0 +1,69 @@
+/datum/species/moth
+	// Shitty moths, why do we need them dammit
+	name = "Mothmen"
+	id = "moth"
+	say_mod = "flutters"
+	default_color = "00FF00"
+	species_traits = list(LIPS, SPECIES_ORGANIC)
+	mutant_bodyparts = list("moth_wings")
+	default_features = list("moth_wings" = "Plain")
+	attack_verb = "slash"
+	attack_sound = 'sound/weapons/slash.ogg'
+	miss_sound = 'sound/weapons/slashmiss.ogg'
+	scream_sound = 'sound/effects/scream_moth.ogg'
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/moth
+	liked_food = VEGETABLES | DAIRY
+	disliked_food = FRUIT | GROSS
+	toxic_food = MEAT | RAW
+	var/next_fly_at_light_attempt = 0
+
+/datum/species/moth/random_name(gender,unique,lastname)
+	if(unique)
+		return random_unique_moth_name(gender)
+
+	var/randname = moth_name(gender)
+
+	if(lastname)
+		randname += " [lastname]"
+
+	return randname
+
+/datum/species/moth/qualifies_for_rank(rank, list/features)
+	if(CONFIG_GET(flag/enforce_human_authority) && (rank in GLOB.command_positions))
+		return FALSE
+	return TRUE
+
+/datum/species/moth/handle_fire(mob/living/carbon/human/H, no_protection = FALSE)
+	..()
+	if(H.dna.features["moth_wings"] != "Punished")
+		if(H.bodytemperature >= 800)//If you reach this, you're in the red zone with the lil +.
+			to_chat(H, "<span class='danger'>Your precious wings burn to a crisp!</span>")
+			H.dna.features["moth_wings"] = "Punished"
+			handle_mutant_bodyparts(H)
+
+/datum/species/moth/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
+	. = ..()
+	if(chem.id == "pestkiller")
+		H.adjustToxLoss(3)
+		H.reagents.remove_reagent(chem.id, REAGENTS_METABOLISM)
+
+/datum/species/moth/check_weakness(obj/item/weapon, mob/living/attacker)
+	if(istype(weapon, /obj/item/melee/flyswatter))
+		return 10 //Flyswatters deal 10x damage to flypeople.
```

Comment:
That's 11x damage.

Reply:
no its not

Reply:
it clearly says 10

Reply:
wait hold on no its just shitcode youre right